### PR TITLE
Update DiagnosticReport documentation and response

### PR DESCRIPTION
**Pull request by daliboz**
Updates for the new May 2015 response fields and required parameter documentation

### CODE REVIEW

File: lib/resources.rb [hunk 1]
```diff
@@ -1453,27 +1453,163 @@ def get_resource(key)
...
+                            {
+                                "system" => "http://hl7.org/fhir/v2/0074",
+                                "code" => "RAD",
+                                "display" => "Radiology"
+                            }
+                        ],
+                        "text" => "Radiology"
+                    },
+                    "diagnosticDateTime" => "2015-08-06T20:03:09.000Z",
+                    "conclusion" => "Report_asdfasdfasdf jjimmmy   Signature Line:***** Preliminary Report *****   Transcribed by: DW      Reason for exam:4061979"
```
Comment (namotron) [inline lib/resources.rb]: Since this is going to be publicly visible, should we clean up some of these values?

Comment (cerbs) [inline lib/resources.rb]: Since this is going to be publicly visible, should we clean up some of these values?

**Comment by MaxPhilips** [comment]
+1

**Comment by jfrhong** [comment]
+1

**Comment by daliboz** [comment]
Updated the one diagnostic report with weird text.

**Comment by dennispatterson** [comment]
:+1:

**Pull request merged**